chains/evm/cli/erc721/deposit.go

```diff
@@ -40,13 +40,13 @@ var depositCmd = &cobra.Command{
 }
 
 func BindDepositCmdFlags(cmd *cobra.Command) {
-	cmd.Flags().StringVar(&Recipient, "recipient", "", "address of recipient")
-	cmd.Flags().StringVar(&Bridge, "bridge", "", "address of bridge contract")
-	cmd.Flags().StringVar(&DestionationID, "destId", "", "destination domain ID")
-	cmd.Flags().StringVar(&ResourceID, "resourceId", "", "resource ID for transfer")
-	cmd.Flags().StringVar(&TokenId, "tokenId", "", "ERC721 token ID")
-	cmd.Flags().StringVar(&Metadata, "metadata", "", "ERC721 metadata")
-	flags.MarkFlagsAsRequired(cmd, "recipient", "bridge", "destId", "resourceId", "tokenId")
+	cmd.Flags().StringVar(&Recipient, "recipient", "", "Recipient address")
+	cmd.Flags().StringVar(&Bridge, "bridge", "", "Bridge contract address")
+	cmd.Flags().StringVar(&DestionationID, "dest-id", "", "Destination domain ID")
```

Comment:
We use full words everywhere else, so I probably wouldn't use `dest` shorthand and would use `destination`.
If we want to use shorthands, viper allows for that.

Reply:
Agree with Matija regarding `destination`.
Regarding shorthands, it can only be on letter, so would avoid using that, because for sure there is a command starting with same letter and that would bring confusion